migrations.json

```diff
@@ -0,0 +1,324 @@
+{
```

Comment:
It seems that this file should not be committed
https://nx.dev/core-features/automate-updating-dependencies#step-3:-cleaning-up